src/prerender_content.js

```diff
@@ -5,11 +5,30 @@ import "./Objs";
 import "./Widgets";
 import "./config";
 import prerenderObjs from "./prerenderContent/prerenderObjs";
+import prerenderSitemap from "./prerenderContent/prerenderSitemap";
 
-const BLACKLIST_OBJ_CLASSES = ["Download", "Image", "Redirect", "Video"];
+const PRERENDER_BLACKLIST_OBJ_CLASSES = [
+  "Download",
+  "Image",
+  "Redirect",
+  "Video",
+];
 
-function prerenderContent() {
-  return prerenderObjs(BLACKLIST_OBJ_CLASSES);
+const SITEMAP_OBJ_CLASSES_WHITELIST = [
```

Comment:
* [ ] Naming inconsistent: `PRERENDER_OBJ_CLASSES_BLACKLIST` maybe? Also check, if the var names in the callee match.